crates/re_types/src/components/point3d.rs

```diff
@@ -221,20 +221,56 @@ impl crate::Loggable for Point3D {
         })?)
     }
 
+    #[allow(unused_imports, clippy::wildcard_imports)]
+    #[inline]
+    fn try_from_arrow(data: &dyn ::arrow2::array::Array) -> crate::DeserializationResult<Vec<Self>>
+    where
+        Self: Sized,
+    {
+        use crate::Loggable as _;
+        use ::arrow2::{array::*, buffer::*, datatypes::*};
+        Ok({
+            let data = data
+                .as_any()
+                .downcast_ref::<::arrow2::array::FixedSizeListArray>()
+                .unwrap();
+            if data.is_empty() {
+                itertools::Either::Left(std::iter::empty())
+            } else {
+                let data = &**data.values();
+                let data = data
+                    .as_any()
+                    .downcast_ref::<Float32Array>()
+                    .unwrap()
+                    .values()
+                    .as_slice();
+                let data2: &[[f32; 3]] = bytemuck::cast_slice(data);
```

Comment:
This is one of the main things we want to generate for fixed-sized-arrays of primitives.